src/structures/GuildChannel.js

```diff
@@ -156,6 +156,14 @@ class GuildChannel extends Channel {
     return [...sorted.values()].indexOf(sorted.get(this.id));
   }
 
+  /**
+   * Checks if the channel is not viewable by users with only the @everyone role.
+   * @type {boolean} true if private, false otherwise.
```

Comment:
This seems quite redundant to me.
```suggestion
   * @type {boolean}
```